docs/scenarios/ebs-pv/README.md

```diff
@@ -0,0 +1,5 @@
+## Migrating a **stateful** OpenShift app with **Amazon EBS PVs**
+
+This scenario walks through Migration of a stateful OpenShift app whose Persistent Volume (PV) claims tie into [Amazon EBS (Elastic Block Storage)](https://aws.amazon.com/ebs/).
+
+
```

Comment:
Let's add a section to each scenario overview stating what PV actions are supported.

For example for ebs-pv we support
 'Copy', usage of ebs snapshots will result in a new PV being created from a snapshot of the PV.
'Move' is not supported.

Reply:
Will add this.

Sidenote:  are we tabling the concept of disconnecting an EBS volume from one instance and connecting it to another?